.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Description of the change**
+
+Clearly and concisely describe the purpose of the pull request. If this PR relates to an existing issue or change proposal, please link to it. Include any other background context that would help reviewers understand the motivation for this PR.
+
+---
+
+**Checklist:**
+
+- [ ] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
```

Comment:
I agree with @f-f that perhaps we don't require a _link_ to this information, but rather just the PR number -- like `Updated return type of function to be an integer (#55)`. We may not need the username either.

Reply:
I'll make that change to my template.

Reply:
How's this?

```
- [ ] Added the change to the changelog's "Unreleased" section with a 
      reference to this PR (e.g. "- Made a change (#22)")
```